.travis.yml

```diff
@@ -5,7 +5,7 @@ language: ruby
 cache: bundler
 sudo: false
 rvm:
-- 2.3.1
+- 2.4.1
```

Comment:
Looks like Chef 14 comes with ruby 2.5 https://docs.chef.io/release_notes.html#what-s-new-in-14-0. Can we go to 2.5?